### PR TITLE
[MCHANGES-434] @Component --> @Inject

### PR DESCRIPTION
**Pull request by elharo**

### CODE REVIEW

File: src/main/java/org/apache/maven/plugins/announcement/AnnouncementMojo.java [hunk 1]
```diff
@@ -471,6 +460,22 @@ public class AnnouncementMojo extends AbstractAnnouncementMojo {
 
     private ChangesXML xml;
 
+    /**
+     * Velocity Component.
+     */
+    private VelocityComponent velocity;
```
Comment (slawekjaranowski) [inline src/main/java/org/apache/maven/plugins/announcement/AnnouncementMojo.java:466]: final?

**Review by slawekjaranowski — COMMENTED** [review]
We can remove plugin execution `plexus-component-metadata`

**Comment by slawekjaranowski** [comment]
If we use `@` and something in commits, descriptions ... GitHub  create a link to such account.

We should escape such words by ` , eg

```
`@Inject`
```

**Comment by elharo** [comment]
I don't see any such links. I think it only autolinks in textboxes like this one, not everywhere

File: src/main/java/org/apache/maven/plugins/announcement/AnnouncementMojo.java (hunk 1 above)
Comment (elharo) [reply src/main/java/org/apache/maven/plugins/announcement/AnnouncementMojo.java]: done

**Comment by elharo** [comment]
> I don't see any such links. I think it only autolinks in textboxes like this one, not everywhere

OK I do see one such link. However I still think we shouldn't change the general git commit message based on how Github will interpret it.

**Review by slawekjaranowski — APPROVED** [review]

**Pull request merged**